tests/appsec/test_blocking_addresses.py

```diff
@@ -114,6 +123,7 @@ def setup_response_header(self):
         self.rsh_req = weblog.get(path="/headers")
 
     @missing_feature(context.library == "java", reason="Happens on a subsequent WAF run")
+    @released(ruby="?")
     def test_response_header(self):
         """can block on server.response.headers.no_cookies"""
```

Comment:
I think we ultimately can have that one as well, we're just missing the address for Rack.